var/spack/repos/builtin/packages/arbor/package.py

```diff
@@ -53,20 +58,23 @@ class Arbor(CMakePackage, CudaPackage):
     )
 
     # https://docs.arbor-sim.org/en/latest/install/build_install.html#compilers
-    conflicts("%gcc@:8.3")
-    conflicts("%clang@:7")
+    conflicts("%gcc@:8")
```

Comment:
we restrict more now?

Reply:
Yes, as of [this PR](https://github.com/arbor-sim/arbor/pull/2017) minimum versions for compilers were bumped. Is that a problem? I asked in the channel, and nobody objected (otherwise I would not have merged this). One reason to bump is to be able to support CUDA 11.

Reply:
Sorry, I was misreading conflicts vs. depends_on.